cbv/management/commands/load_all_django_versions.py

```diff
@@ -0,0 +1,22 @@
+import os
+import re
+
+from django.conf import settings
+from django.core.management import call_command, BaseCommand
+
+
+class Command(BaseCommand):
+    """Load the Django project fixtures and all version fixtures"""
+
+    def handle(self, **options):
+        fixtures_dir = os.path.join(settings.DIRNAME, 'cbv', 'fixtures')
+        self.stdout.write('Loading project.json')
+        call_command('loaddata', 'cbv/fixtures/project.json')
+        version_fixtures = [re.match(r'((?:\d+\.){2,3}json)', filename) for filename in os.listdir(fixtures_dir)]
+        for match in version_fixtures:
+            try:
+                fixture = match.group()
+            except AttributeError:
+                continue
+            self.stdout.write('Loading {}'.format(fixture))
+            call_command('loaddata', 'cbv/fixtures/{}'.format(fixture))
```

Comment:
Yeah I thought about it but decided that `loaddata` was probably handling turning it into a valid path anyway. Maybe it doesn't, but I'd never seen mention of having to call it differently on Windows or whatever.